features/shop/promotion/applying_promotion_coupon/applying_promotion_coupon.feature

```diff
@@ -10,7 +10,7 @@ Feature: Applying promotion coupon
         And the store has promotion "Christmas sale" with coupon "SANTA2016"
         And this promotion gives "$10.00" discount to every order
 
-    @todo @ui @api
+    @api @ui @mink:chromedriver
```

Comment:
According to https://github.com/Sylius/Sylius/pull/16820#issuecomment-2326095118, is there a need to tag these scenarios with `@mink:chromedriver`?

Reply:
Yeah, the coupon field is no-live. But still it is inside live component and it is submitted with button (via javascript under the hood). So, yes unfortunately it needs to be tested with JS 😞